wiki/en/Running-a-Server.md

```diff
@@ -100,6 +89,10 @@ If you want to run a Server on a Raspberry Pi, you will need to [compile from so
 1. Reload the systemd files `sudo systemctl daemon-reload` and restart the headless Server: `sudo systemctl restart jamulus-headless`
 1. Check all is well with `systemctl status jamulus-headless` (hit `q` to get back to the command prompt).
 
+_To upgrade your Server, just repeat the steps above._
+
+If you are running an unregistered server behind a home internet connection, you might need to enable [port forwarding](port-forwarding) as described below.
```

Comment:
Is the target for the link right?   (No `#`.)

Reply:
"Enable the headless Server process via systemd" - is that actually necessary any more with the .deb? I think it might be left over from when you had to roll your own unit file - does the installer not enable it for you?

Reply:
Fixed the initial error. And I think enabling is necessary.